cmd/libsnap-confine-private/mount-opt.c

```diff
@@ -288,6 +291,21 @@ void sc_do_mount(const char *source, const char *target,
 		errno = saved_errno;
 		die("cannot perform operation: %s", mount_cmd);
 	}
+	return true;
+}
+
+void sc_do_mount(const char *source, const char *target,
+		 const char *fs_type, unsigned long mountflags,
+		 const void *data)
+{
+	(void)sc_do_mount_ex(source, target, fs_type, mountflags, data, false);
```

Comment:
Why the cast to void?

Reply:
To explicitly indicate that we discard the return value and we are not just not aware of one.